vm_image/init

```diff
@@ -5,6 +5,10 @@
 
 set -x
 
+# Setup the 'neon-test' cgroup:
+whoami
+mkdir /sys/fs/cgroup/neon-test
```

Comment:
does this leak into a production environment? does the whoami need to stay?

Reply:
good catch - the `whoami` is just for debugging. I'll make a note to remove it once I'm done testing stuff.

Reply:
Done